sourced/ml/cmd_entries/repos2df.py

```diff
@@ -14,17 +14,17 @@ def repos2df_entry(args):
     extractors = [__extractors__[s](
         args.min_docfreq, **__extractors__[s].get_kwargs_fromcmdline(args))
         for s in args.feature]
+    document_column_name = "repository_id"
```

Comment:
Would be nice to declare a constant somewhere. E.g. new file `engine_constants.py` or just `engine.py` which already exists